nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -2049,4 +2049,70 @@ public void checkVersionSerialization(int version) {
           "Could not read serialization version at path: " + serializationVersionPath, e);
     }
   }
+
+  @Test
+  public void varArgsWithTypeUseAnnotationMethodSerializationTest() {
+    SerializationTestHelper<ErrorDisplay> tester = new SerializationTestHelper<>(root);
+    tester
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:SerializeFixMetadata=true",
+                "-XepOpt:NullAway:FixSerializationConfigPath=" + configPath))
+        .addSourceLines(
+            // toString() call on method symbol will serialize the annotation below which should not
+            // be present in string representation fo a method signature.
+            "com/uber/Custom.java",
+            "package com.uber;",
+            "import java.lang.annotation.Target;",
+            "import static java.lang.annotation.ElementType.TYPE_USE;",
+            "@Target({TYPE_USE})",
+            "public @interface Custom { }")
+        .addSourceLines(
+            "com/uber/Test.java",
+            "package com.uber;",
+            "import java.util.*;",
```

Comment:
Even in tests, I'd prefer no star imports if possible.

Reply:
Sure, fixed it in 73f1a850d94c9a9d1cc47090b34ba54a6efb3ef3